doc/manual/50-feature-details.en.md

```diff
@@ -884,6 +884,59 @@ The `replication-status` command allows
 you to get information about any streaming client attached to the
 managed server, in particular hot standby servers and WAL streamers.
 
+### Configuration Models
+
+Configuration models define a set of overrides for configuration options. These
+overrides can be applied to Barman servers which are part of the same cluster as
+the config models. They can be handy to handle clustered environments, so you
+can change the configuration of a Barman server in response to failover events,
+for example.
+
+As an example, let's say you have a PostgreSQL cluster with the following nodes:
+
+* `pg-node-1`: primary
+* `pg-node-2`: standby
+* `pg-node-3`: standby
+
+Assume you are backing up from the primary node, and have a configuration which
+include the following options:
+
+```ini
+[my-barman-server]
+cluster = my-cluster
+conninfo = host=pg-node-1 user=barman database=postgres
+streaming_conninfo = host=pg-node-1 user=streaming_barman
+; other options...
+```
+
+You could, for example, have a configuration model for that cluster as follows:
+
+```ini
+[my-barman-server:backup-from-pg-node-2]
+cluster = my-cluster
+model = true
+conninfo = host=pg-node-2 user=barman
```

Comment:
It might be worth keeping the conninfo the same apart from the host, just in case a user misinterprets that the `database=postgres` is omitted from conninfo here because it's present in the conninfo in the server configuration.

```suggestion
conninfo = host=pg-node-2 user=barman database=postgres
```

Reply:
Makes sense!